src/main/java/org/example/Board.java

```diff
@@ -1,76 +1,98 @@
 package org.example;
+import lombok.Getter;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 
+@Getter
 public class Board {
-    private List<Integer> intList ;
-    private static final String RED = "\u001B[31m";
-    private static final String GREEN = "\u001B[32m";
-    private static final String WHITE = "\u001B[0m";
-    private char[][] board ;
+  private List<Integer> emptyPositions;
+  private static final char X_SYMBOL = 'X';
+  private static final char O_SYMBOL = 'O';
+  private static final String SEPARATOR = " | ";
+  private static final String RED = "\u001B[31m";
+  private static final String GREEN = "\u001B[32m";
+  private static final String WHITE = "\u001B[0m";
+  private char[][] board;
 
-    public char[][] getBoard() {
-        return board;
-    }
+  private void print(String text) {
+    System.out.print(text);
+  }
+
+  public Board() {
+    board = new char[3][3];
+    emptyPositions = new ArrayList<>();
+  }
 
-    public List<Integer> getIntList() {
-        return intList;
+  public void initializeBoard() {
+    int positionCount = 1;
+    for (int i = 0; i < 3; i++) {
+      for (int j = 0; j < 3; j++) {
+        board[i][j] = (char) (positionCount + 48);
+        positionCount++;
+      }
     }
+    prepareComputerMoves();
+  }
 
+  public void prepareComputerMoves() {
 
-    public Board()
-    {
-         board = new char[3][3];
-          intList = new ArrayList<>();
-    }
-    public  void initializeBoard() {
-        int a = 1;
-        for (int i = 0; i < 3; i++) {
-            for (int j = 0; j < 3; j++) {
-                board[i][j] = (char) (a + 48);
-                a++;
-            }
-        }
-        prepareComputerMoves();
-    }
-    public void prepareComputerMoves()  {
+    emptyPositions.clear();
+    emptyPositions.addAll(Arrays.asList(1, 2, 4, 3, 5, 6, 7, 8, 9));
+    Collections.shuffle(emptyPositions);
```

Comment:
Here if you want to implement nXn TicTacToe, hen what will be your approach? As you are adding constant numbers and shuffling them

Reply:
After discussing with the colleagues, I am thinking to apply the NXN board and strategy changes in new branch so there i will incorporate all the mentioned changes.

Reply:
> Here if you want to implement nXn TicTacToe, hen what will be your approach? As you are adding constant numbers and shuffling them

I will take the numbers from 1 to NXN and will shuffle it

Reply:
Yaa, But you are manually entering all the numbers from 1 to 9 (3X3). Now let's say you have 10X10 board will you enter all 100 numbers manually?